sql/atxs/atxs.go

```diff
@@ -794,3 +794,18 @@ func IterateForGrading(
 	}
 	return nil
 }
+
+func PoetProofRef(ctx context.Context, db sql.Executor, id types.ATXID) (types.PoetProofRef, error) {
+	var blob sql.Blob
+	if err := LoadBlob(ctx, db, id.Bytes(), &blob); err != nil {
+		return types.PoetProofRef{}, fmt.Errorf("getting blob for %s: %w", id, err)
+	}
+
+	// TODO: decide about version based on publish epoch
```

Comment:
@fasmat I think it would be the easiest to put another column `version` in the `atxs_blob` table. Wdyt?

Reply:
Hm... I don't think we need version info in the DB. We can fetch the publish epoch of the ATX with it's ID and there will be some epoch n which is the last epoch that uses V1 and every ATX published in epoch n+1 or later will be V2.

Reply:
Yes, but the problem is that you must pass the "v2-since-epoch-N" configuration value to all functions that will decode the blob. Whereas with the `version` column it's not required - the function can decide by looking at the value of `version`.